tiles/src/main/java/com/protomaps/basemap/layers/Places.java

```diff
@@ -98,18 +94,18 @@ public void processNe(SourceFeature sf, FeatureCollector features) {
       var feat = features.point(this.name())
         .setAttr("name", sf.getString("name"))
         .setAttr("pmap:min_zoom", minZoom)
+        // We subtract 1 to achieve intended compilation balance vis-a-vis 256 zooms in NE and 512 zooms in Planetiler
+        .setZoomRange((int) minZoom - 1, 6)
         .setAttr("pmap:kind", kind)
         .setAttr("pmap:kind_detail", kindDetail)
         .setAttr("population", population)
         .setAttr("pmap:population_rank", populationRank)
         .setAttr("wikidata_id", sf.getString("wikidata"))
-        // We subtract 1 to achieve intended compilation balance vis-a-vis 256 zooms in NE and 512 zooms in Planetiler
-        .setZoomRange((int) minZoom - 1, themeMaxZoom)
-        .setBufferPixels(128)
+        .setBufferPixels(64)
+        .setPointLabelGridPixelSize(7, 64) // 64 pixels is 1/4 the tile, so a 4x4 grid
```

Comment:
Let's unify all cell sizes on 64 (4x4 grid) for now - you can tweak this to taste.

Reply:
This now works as expected. I tweaked the settings a little.

In future it may make more sense to do 3x3 grid with larger number of children... But that should vary by zoom and I don't see how to configure that in Planetiler...